### ISSUE TITLE
Row border don't appear by default on Firefox

### ISSUE THREAD

**Issue by Jenselme** [KevinVandy/material-react-table#320]
### material-react-table version

1.5.7

### react & react-dom versions

17.0.2

### Describe the bug and the steps to reproduce it

On Firefox (and Firefox only), the border below each rows are not render in the default configuration. The border becomes visible when the row is hovered.

Adding this style to the props fixes the issues:
```js
      muiTableBodyProps={{
        sx: {
          tr: {
            backgroundColor: "unset",
          },
        },
      }}
```
However, it does changes how the body looks: a shadow appears below the header. I don't think I'm competent enough on style to provide any help.

### Minimal, Reproducible Example - (Optional, but Recommended)

You can check out any of the official examples to see the issue: https://www.material-react-table.com/docs/examples/basic

### Screenshots or Videos (Optional)

So in Chrome, the default table looks like this:
![image](https://user-images.githubusercontent.com/6643536/212290326-68fbdcb4-2d91-4933-ba83-e339a5a6c07e.png)
But looks like this in Firefox:
![image](https://user-images.githubusercontent.com/6643536/212290409-905cf9a0-7e6f-49e1-a73d-8d09d3440a41.png)

### Do you intend to try to help solve this bug with your own PR?

No, because I do not know how

### Terms

- [X] I understand that if my bug cannot be reliably reproduced in a debuggable environment, it will probably not be fixed and this issue may even be closed.

**Comment by KevinVandy** [comment]
Try this instead:

```js
muiTableProps={{
  sx: {
    borderCollapse: 'separate',
  },
}}
```

I'm doing more testing to see if this should just be a default style in the library internally. Seems to only be Firefox that renders it with this bug

**Comment by KevinVandy** [comment]
This is now included in v1.5.9

**Comment by Jenselme** [comment]
That was fast :star_struck: I just made the update on my project and everything works out of the box on firefox. Thank you very much!
